spec/support/dummy_server/servlet.rb

```diff
@@ -148,6 +148,16 @@ def do_#{method.upcase}(req, res)
       res.body   = req.body
     end
 
+    get "/uri_normalizer/ABC" do |_req, res|
+      res.status = 200
+      res.body   = "default normalizer"
+    end
+
+    get "/uri_normalizer/custom" do |_req, res|
+      res.status = 200
+      res.body   = "custom normalizer"
+    end
```

Comment:
just add one endpoint for testing. As simple as:

``` ruby
get "/hello;world" do |_req, res|
  res.status = 200
  res.body   = "hello world"
end
```

Default normalizer will hit 404 in this case, and custom should get "hello world".